### ISSUE TITLE
use go channels to more quickly figure out closest region

### ISSUE THREAD

**Issue by anuaimi** [eezhee/eezhee#66]
right now we do this serially.  each time the command is run.  Should be done in parallel using go channels.

**Comment by anuaimi** [comment]
isn't there a version of this for the Vultr provider?  just need to generalize

**Comment by anuaimi** [comment]
duplicate of #71
